example/src/Components/SettingsTable/index.tsx

```diff
@@ -54,6 +54,8 @@ const SettingsTable: FC<SettingsTableProps> = ({ transformationFunction }) => {
     const storedSettings = await getItem();
     if (storedSettings) {
       setSettings(JSON.parse(storedSettings));
+    } else {
+      setSettings(DEFAULT_SETTINGS_DATA);
```

Comment:
Since the DEFAULT_SETTINGS_DATA was the initial value `settings`, I don't see how this would help.
We used to not setSettings in case of a falsy  storedSettings which ends up in keeping the defaults.

Reply:
It was not set as default value for settings, I just checked.

I checked this flow with debugger, and without these default values being set, when we change one setting (that requires another fragment like CRI) it is incorrectly saved as incomplete settings. 

So when I comment out this line I've added - then we observe "Cannot read property 'isEnabled' of undefined" as it tries to use some settings that haven't been initialised.